### ISSUE TITLE
deferToThreadPool should be in __all__

### ISSUE THREAD

**Issue by twisted-trac** [twisted/twisted#4264]
|<img alt="allenap's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">| allenap reported|
|-|-|
|Trac ID|trac#4264|
|Type|defect|
|Created|2010-02-09 15:31:27Z|

I think it would make sense to add deferToThreadPool to twisted.internet.threads.__all__. It's a useful function... and Launchpad's importfascist complains when I use it :)

To make the decision even easier, I've attached a patch.

Gavin.

Attachments:

* [deferToThreadPool-in-__all__.patch](https://raw.githubusercontent.com/twisted/twistedmatrix.com-trac-attachments/trunk/ticket/cd6/cd6a52531215682b6af7a71cdf9f8770ce991a9e/799a7e28cd000b33584b7f4e2a7792bab6e55396.patch) (401 bytes) - added by allenap on 2010-02-09 15:31:57Z - Patch to add deferToThreadPool to __all__

<details><summary>Searchable metadata</summary>

```
trac-id__4264 4264
type__defect defect
reporter__allenap allenap
priority__normal normal
milestone__ 
branch__ 
branch_author__ 
status__closed closed
resolution__fixed fixed
component__core core
keywords__ 
time__1265729487000000 1265729487000000
changetime__1265739803000000 1265739803000000
version__None None
owner__ 

```
</details>

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_1"></a>|@exarkun set owner to @exarkun<br>@exarkun set status to `assigned`|
|-|-|

Thanks for catching this.  It looks like it was just an oversight to not have this in `__all__`.  Applying your patch.  Thanks.

**Comment by twisted-trac** [comment]
|[<img alt="exarkun's avatar" src="https://avatars.githubusercontent.com/u/254565?s=50" width="50" height="50">](https://github.com/exarkun)<a name="note_2"></a>|@exarkun set status to `closed`|
|-|-|

(In [28139]) Apply deferToThreadPool-in-__all__.patch adding a missing name to `twisted.internet.threads.__all__`

Author: allenap
Reviewer: exarkun
Fixes: [#4264](https://github.com/twisted/twisted/issues/4264)

The recently added `deferToThreadPool` method was not named in the `__all__`
of `twisted.internet.threads`.  Add it.
Also bump the clearly out-of-date copyright date to 2010.

**Comment by twisted-trac** [comment]
|<img alt="Automation's avatar" src="https://avatars.githubusercontent.com/u/0?s=50" width="50" height="50">|Automation removed owner|
|-|-|
